app/routes.py

```diff
@@ -1,2 +1,221 @@
-from flask import Blueprint
+from app import db 
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify, Flask 
+from datetime import datetime 
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import desc, asc 
+import os
+from dotenv import load_dotenv
+import requests
 
+load_dotenv()
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+# TASK ROUTES
+@task_bp.route("", methods=["POST"])
+def create_task():
+    """Create a task for the database"""
+    request_body = request.get_json()
+
+    if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+        return make_response({"details": "Invalid data"}, 400)
+    new_task = Task(title=request_body["title"],
+                    description=request_body["description"], 
+                    completed_at=request_body["completed_at"])
+    
+    db.session.add(new_task)
+    db.session.commit()
+    return jsonify({"task": new_task.to_json()}), 201
+
+@task_bp.route("", methods=["GET"])
+def get_all_tasks():
+    """Get multiple tasks per request"""
+    tasks_ordered = request.args.get("sort")
+
+    if not tasks_ordered:
+        tasks = Task.query.all()
+    elif tasks_ordered == "asc":
+        tasks = Task.query.order_by(asc(Task.title))
+    elif tasks_ordered == "desc":
+        tasks = Task.query.order_by(desc(Task.title))
+
+    hold_tasks = []
```

Comment:
`hold_tasks` is kind of an odd name.  I probably would have gone with something like `task_dicts`.